Android/app/src/main/AndroidManifest.xml

```diff
@@ -2,6 +2,11 @@
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
     xmlns:tools="http://schemas.android.com/tools">
 
+    <uses-permission android:name="android.permission.CAMERA" />
```

Comment:
Why camera permission?

Reply:
Good catch. I was copying the Blockstream Green permissions

Reply:
Feel free to keep the internet permission; remove just the camera stuff

Reply:
Done